unity-renderer/Assets/UIComponents/Scripts/Components/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+public class AvatarSlotComponentView : BaseComponentView, IPointerClickHandler, IAvatarSlotComponentView, IComponentModelConfig<AvatarSlotComponentModel>
```

Comment:
can you please move all the clases to the backpack v2 folder and add the `DCL.Backpack` namespace aswell